packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/comparators/workspace-field.comparator.ts

```diff
@@ -108,10 +114,6 @@ export class WorkspaceFieldComparator {
       const findField = (
         field: ComputedPartialFieldMetadata | FieldMetadataEntity,
       ) => {
-        if (field.isCustom) {
```

Comment:
I don't understand what this was for, we were comparing name with ids (fieldName is an id)